src/components/Bounties/ClaimChildBountyButton.tsx

```
@@ -0,0 +1,156 @@
+				threshold: data?.threshold || 0
+			});
+		}
+	};
+
+	useEffect(() => {
+		handleMultisigAddress();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [api, apiReady, network, multisigAssociatedAddress, loginAddress]);
+
+	const handleClaim = async () => {
+		if (!api || !apiReady || bountyIndex == null || isNaN(bountyIndex)) return;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)